### PR TITLE
feature(form):fluentValidation support dependencyInjection

### PR DESCRIPTION
**Pull request by 15168440402**

### CODE REVIEW

**Comment by doddgu** [comment]
@capdiem PTAL

File: src/Component/BlazorComponent/Extensions/DependencyInjection/ServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -18,8 +18,32 @@ public static IServiceCollection AddBlazorComponent(this IServiceCollection serv
             services.TryAddSingleton<IComponentIdGenerator, GuidComponentIdGenerator>();
             CultureInfo.DefaultThreadCurrentUICulture = CultureInfo.CurrentCulture;
             services.AddSingleton<IComponentActivator, AbstractComponentActivator>();
+            services.AddFluentValidation();
```
Comment (capdiem) [inline src/Component/BlazorComponent/Extensions/DependencyInjection/ServiceCollectionExtensions.cs:21]: 可以使用官方的DI吗？https://docs.fluentvalidation.net/en/latest/di.html

**Review by capdiem — CHANGES_REQUESTED** [review]

**Review by capdiem — APPROVED** [review]
LGTM. @doddgu

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**